src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -32,7 +32,9 @@
       <div v-if="showErrorSummary" class="people-roles-invalid-message">
         <span>
           <v-icon color="error">mdi-information-outline</v-icon>
-          <span class="error-text"> This step is not complete. </span>
+          &nbsp;
+          <span class="error-text">This step is not complete.</span>
+          &nbsp;
```

Comment:
I did this to be explicit that we need a space between the icon, text and link. (Otherwise, leading/trailing spaces within a span can look like a typo and someone might delete them to clean up, and break this.)